tests/Fixer/Strict/DeclareStrictTypesFixerTest.php

```diff
@@ -141,11 +155,11 @@ public function provideMessyWhitespacesCases()
     {
         return [
             [
-                "<?php declare(strict_types=1);\r\nphpinfo();",
+                "<?php\r\ndeclare(strict_types=1);\r\n\tphpinfo();",
```

Comment:
> it will fail on any integration usage that are using this rule.

Actually, having a think about it: it shouldn't fail with integration usage. Not unless the integration is using the fixer on code that has `declare(strict_types=1)` absent. On code that has already been fixed (or that where `declare(strict_types=1)` is already present), the `DeclareStrictTypesFixer` won't touch it. The repositioning fixer doesn't run by default anyway, so can probably leave the default as is?

Only thing that has changed RE this is that iff code lacks `declare(strict_types=1)` then it'll insert it on the next line (instead of on the same one).